addons/survey/wizard/survey_invite_views.xml

```diff
@@ -5,27 +5,30 @@
             <field name="name">survey.invite.view.form</field>
             <field name="model">survey.invite</field>
             <field name="arch" type="xml">
-                <form string="Compose Email" class="o_mail_composer_form">
+                <form string="Compose Email" class="o_mail_composer_form" style="padding-bottom: 0px;">
```

Comment:
We should avoid inline style in XML definition.

If you need custom styling, you can go for a specific class.
(But it's strange you need one).
(Alternatively, pb-0 could be sufficient).